File: VB/DXExample.Module/MySimpleUser.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.Xpo

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation

Namespace DXExample.Module
	Public Class MySimpleUser
		Inherits SimpleUser
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		<Association("BOs-Users"), Browsable(False)> _
		Public ReadOnly Property BOs() As XPCollection(Of BaseClass)
			Get
				Return GetCollection(Of BaseClass)("BOs")
			End Get
		End Property
	End Class

End Namespace

File: CS/DXExample.Module/MySimpleUser.cs
using System;
using System.ComponentModel;

using DevExpress.Xpo;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;

namespace DXExample.Module {
    public class MySimpleUser : SimpleUser {
        public MySimpleUser(Session session) : base(session) { }
        [Association("BOs-Users")]
        [Browsable(false)]
        public XPCollection<BaseClass> BOs {
            get { return GetCollection<BaseClass>("BOs"); }
        }
    }

}
